vulture/core.py

```diff
@@ -491,40 +491,36 @@ def generic_visit(self, node):
 
 
 def _parse_args():
-    def csv(option, _, value, parser):
-        setattr(parser.values, option.dest, value.split(','))
-    usage = """\
-usage: %prog [options] PATH [PATH ...]
-
-Paths may be Python files or directories. For each directory vulture
-analyzes all contained *.py files.
-"""
+    def csv(exclude):
+        return [str(excluded) for excluded in exclude.split(',')]
```

Comment:
I think we can just `return exclude.split()`.

Reply:
Yes and yes. 😲